packages/backend/src/api/1/handlers/auth.js

```diff
@@ -1,3 +1,48 @@
+import crypto from 'crypto';
 import express from 'express';
+import passport from 'passport';
 
-module.exports = express.Router();
+const router = express.Router();
+
+router.get(
```

Comment:
Ain't really what I was asking.
I'll explain in PR comments more concise, but:
separate the endpoints so they're
/reddit
and
/reddit/callback